server/api/todos/index.get.js

```diff
@@ -0,0 +1,32 @@
+import { Todo } from "@/server/models/todo";
+import { todoOutput } from "@/schemas/todo";
+
+// eslint-disable-next-line no-undef
+export default defineEventHandler(async () => {
+    try {
+        const todos = await Todo.findAll({
+            where: {
+                userId: 1,
+            },
+        });
+        let todoArray = [];
+        for (const item of todos) {
+            const todo = {
+                id: item.id,
+                title: item.title,
+                completed: item.completed,
+            };
+
+            const outputResult = todoOutput.safeParse(todo);
+            if (outputResult.success === false) {
+                console.log({ success: false, error: outputResult.error.issues });
+            } else {
+                todoArray.push(outputResult.data);
+                continue;
```

Comment:
Ton `continue` ici n'est pas utile car il n'y a plus d'instruction après ton `else`, donc l'itération sera forcément terminée.